functions/v2/label_gce_instance/main_test.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import re
+import uuid
+
+import backoff
+
+from cloudevents.http import CloudEvent
+
+from google.cloud import compute_v1
+
+import main
+
+
+PROJECT = os.getenv('GCLOUD_PROJECT')
+ZONE = os.getenv('FUNCTIONS_COMPUTE_ZONE')
```

Comment:
nit - for consistency with @savijatv 's earlier suggestion, consider changing this to `INSTANCE_ZONE` throughout

Reply:
Done.